_data/site.yaml

```diff
@@ -98,6 +98,13 @@ searchgov:
   # If you do not want to present search suggestions, set this value to false.
   suggestions: true
 
+  # Used for generating baseurls (important for sitemap.xml to work properly)
+  # Replace prod and staging values with your production URL and your Pages preview URL
+  domains:
+    prod: replaceMe.gov
+    staging:  federalist-replaceMe.sites.pages.cloud.gov/preview/yourOrgName/yourRepoName
+    local: http://localhost:8080
```

Comment:
does it matter that local is the only one with the protocol?

Reply:
I don't know! The URLs are going to vary depending on, I think, whether the site is served at an apex domain or not, though localhost will likely always be accessible at http. We could remove it for consistency if we like, though. This feels like a stylistic issue for the pages team to weigh in on.